src/components/Loans/OpenLoans/components/OpenLoansWithStaticData/OpenLoansWithStaticData.js

```diff
@@ -176,13 +180,22 @@ class OpenLoansWithStaticData extends React.Component {
     const { resources } = this.props;
     const accounts = get(resources, ['loanAccount', 'records'], []);
     const accountsLoan = accounts.filter(a => a.loanId === loan.id) || [];
+    const suspendedStatus = accountsLoan.filter(a => a?.paymentStatus?.name === refundClaimReturned.PAYMENT_STATUS) || [];
+    const suspendedMessage = (suspendedStatus.length > 0) ? 'Suspended' : '';
     let amount = 0;
 
     accountsLoan.forEach(a => {
       amount += parseFloat(a.amount);
     });
 
-    return (amount === 0) ? '-' : amount.toFixed(2);
+    return (amount === 0) ? '-' :
+    <SafeHTMLMessage
+      id="ui-users.loans.details.accounts.suspended"
+      values={{
+        amount,
+        suspended: suspendedMessage
+      }}
+    />;
```

Comment:
FYI `<SafeHTMLMessage>` is no longer necessary as of `@folio/stripes` `v6`; see folio-org/stripes-core/pull/953.

Reply:
Unfortunately, I don't think this translation works as intended; it will _always_ display the value `Suspended`. Instead, you can simply use
```
<FormattedMessage id="ui-users.loans.details.accounts.suspended" values={{ amount }} />
```
and a translation value like
```
"loans.details.accounts.suspended": "{amount} suspended",
```

Reply:
@zburke  Yes, I had not contemplated the translation , thank you for noticing that, I fixed it.